mediathread/templates/courseaffils/course_update_form.html

```diff
@@ -1,49 +1,61 @@
+                <h3 class="card-title">Composition Visibility</h3>
+                {% bootstrap_field form.publish_to_world form_group_class='font-weight-bold mb-2' %}
+            </div>
         </div>
-        <div class="well">
-            <h3>Item &amp; Selection Visibility</h3>
-            {% bootstrap_field form.see_eachothers_items %}
-            {% bootstrap_field form.see_eachothers_selections %}
-            <span class="help-block">
-                Turning off item or selection visibility hides
-                individual items or selections from all collections
-                views. Students can still view an item or selection
-                embedded in a visible composition. Faculty always can
-                view all items and selections. These are both on by
-                default.
-            </span>
+        <div class="card bg-light mb-3">
+            <div class="card-body">
+                <h3 class="card-title">Item &amp; Selection Visibility</h3>
+                {% bootstrap_field form.see_eachothers_items form_group_class='font-weight-bold mb-2' %}
+                {% bootstrap_field form.see_eachothers_selections form_group_class='font-weight-bold mb-2' %}
+                <p class="card-text small">
+                    Turning off item or selection visibility hides
+                    individual items or selections from all collections
+                    views. Students can still view an item or selection
+                    embedded in a visible composition. Faculty always can
+                    view all items and selections. These are both on by
+                    default.
+                </p>
+            </div>
         </div>
         {% if user.is_staff %}
-            <div class="well">
-                <h3>Item Download</h3>
-                {% bootstrap_field form.allow_item_download %}
+            <div class="card bg-light mb-3">
+                <div class="card-body">
+                    <h3 class="card-title">Item Download</h3>
+                    {% bootstrap_field form.allow_item_download form_group_class='font-weight-bold mb-2' %}
+                </div>
             </div>
-            <div class="well">
-                <h3>Allow Roster Management</h3>
-                {% bootstrap_field form.allow_roster_changes %}
+            <div class="card bg-light mb-3">
+                <div class="card-body">
+                    <h3 class="card-title">Allow Roster Management</h3>
+                    {% bootstrap_field form.allow_roster_changes form_group_class='font-weight-bold mb-2' %}
+                </div>
             </div>
         {% endif %}
-        <div class="form-group">
-            <button class="btn btn-primary" type="submit">Save</button>
-            <a href="." class="btn btn-default" role="button">Cancel</a>
+        <div class="form-group float-right">
+            <button class="btn btn-outline-primary" type="submit">Save</button>
```

Comment:
For consistency, let's flip the Save & Cancel on all the dashboard pages. On the page, these buttons appear as `[Save] [Cancel]'. They should appear as `[Cancel] [Save]`.

How're you feeling about the `outline` buttons? I feel like they may be too subtle?

Reply:
Ah right!
Yes on some situations I like them but I also feel like the more prominent primary buttons might work best for Settings pages